### PR TITLE
Add Personal data pages to identity scaffolder content.

### PR DESCRIPTION
**Pull request by prafullbhosale**
based on changes in aspnet/Identity/pull/1559

cc @danroth27

### CODE REVIEW

**Comment by prafullbhosale** [comment]
> Looks fine, is this generated by copying the files over or is done via a hand port?

I hand ported the changes from the PR. Before merging will verify if the generated content matches the content in aspnet/identity

**Comment by HaoK** [comment]
Is there any plan to make this more automated in the future, or will you be hand porting all changes as they are made to the ui package?

File: src/VS.Web.CG.Mvc/Templates/Identity/Pages/Account/Manage/Account.Manage.DownloadPersonalData.cs.cshtml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            _logger.LogInformation("User with ID '{UserId}' asked for their personal data.", _userManager.GetUserId(User));
+
+            // Only include personal data for download
+            var personalData = new Dictionary<string, string>();
+            personalData.Add("UserId", await _userManager.GetUserIdAsync(user));
+            personalData.Add("UserName", await _userManager.GetUserNameAsync(user));
+            personalData.Add("Email", await _userManager.GetEmailAsync(user));
+            personalData.Add("EmailConfirmed", (await _userManager.IsEmailConfirmedAsync(user)).ToString());
+            personalData.Add("PhoneNumber", await _userManager.GetPhoneNumberAsync(user));
+            personalData.Add("PhoneNumberConfirmed", (await _userManager.IsEmailConfirmedAsync(user)).ToString());
```
Comment (seancpeters) [inline src/VS.Web.CG.Mvc/Templates/Identity/Pages/Account/Manage/Account.Manage.DownloadPersonalData.cs.cshtml]: This _userManager call doesn't match the dictionary key.

Comment (prafullbhosale) [reply src/VS.Web.CG.Mvc/Templates/Identity/Pages/Account/Manage/Account.Manage.DownloadPersonalData.cs.cshtml]: @HaoK, I am assuming this isn't intentional and needs to fixed in https://github.com/aspnet/Identity/blob/dev/src/UI/Areas/Identity/Pages/Account/Manage/DownloadPersonalData.cshtml.cs#L45 as well.

Comment (HaoK) [reply src/VS.Web.CG.Mvc/Templates/Identity/Pages/Account/Manage/Account.Manage.DownloadPersonalData.cs.cshtml]: Yeah its not, I'll fix it in UI as well thanks

**Pull request merged**